nginx/files/security.conf

```diff
@@ -1,2 +1,5 @@
 # Security tweaks for nginx
 server_tokens off;
+
+# Cache SSL Sessions to improve performance and security
+ssl_session_cache shared:SSL:50m;
```

Comment:
Zie: https://www.ssllabs.com/ssltest/analyze.html?d=admin.acceptance.mainplus.mainflow.nl&hideResults=on
Volgens mij kan je dit niet uit de testresultaten halen, is toch echt een server instelling waar de client niks van zal merken.